ci/meta.yml

```diff
@@ -89,3 +89,117 @@ meta:
+        run:
+          path: sh
+          args:
+            - -exc
+            - |
+              terraform workspace show
+              terraform init
+              terraform output --json > ../terraform-output-internal-compute/outputs.json
+          dir: aws-internal-compute
+        inputs:
+          - name: aws-internal-compute
+        outputs:
+          - name: terraform-output-internal-compute
+      params:
+        TF_CLI_ARGS_plan: -lock-timeout=300s
+        TF_INPUT: "false"
+        AWS_ACCESS_KEY_ID: ((ci.aws_access_key_id))
+        AWS_SECRET_ACCESS_KEY: ((ci.aws_secret_access_key))
+    terraform-output-ingest:
+      task: terraform-output-ingest
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((terraform.repository))
+            version: ((terraform.version))
+            tag: ((terraform.version))
+        run:
+          path: sh
+          args:
+            - -exc
+            - |
+              terraform workspace show
+              terraform init
+              terraform output --json > ../terraform-output-ingest/outputs.json
+          dir: aws-ingestion
+        inputs:
+          - name: aws-ingestion
+        outputs:
+          - name: terraform-output-ingest
+      params:
+        TF_CLI_ARGS_plan: -lock-timeout=300s
+        TF_INPUT: "false"
+        AWS_ACCESS_KEY_ID: ((ci.aws_access_key_id))
+        AWS_SECRET_ACCESS_KEY: ((ci.aws_secret_access_key))
+    terraform-output-sdx:
+      task: terraform-output-sdx
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((terraform.repository))
+            version: ((terraform.version))
+            tag: ((terraform.version))
+        run:
+          path: sh
+          args:
+            - -exc
+            - |
+              terraform workspace show
+              terraform init
+              terraform output --json > ../terraform-output-sdx/outputs.json
+          dir: aws-sdx
+        inputs:
+          - name: aws-sdx
+        outputs:
+          - name: terraform-output-sdx
+      params:
+        TF_CLI_ARGS_plan: -lock-timeout=300s
+        TF_INPUT: "false"
+        AWS_ACCESS_KEY_ID: ((ci.aws_access_key_id))
+        AWS_SECRET_ACCESS_KEY: ((ci.aws_secret_access_key))
+    synthetic-data-ingestion:
+      task: synthetic-data-ingestion
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((docker-python-boto-behave.repository))
+            version: ((docker-python-boto-behave.version))
+            tag: ((docker-python-boto-behave.version))
+        run:
+          dir: aws-ingestion
+          path: sh
+          args:
+            - -exc
+            - |
+              export AWS_ACC=`cat ../terraform-config/terraform.tf | grep -i development[[:blank:]] | awk '{print $3}'`
```

Comment:
That's a bit brittle, and hard-codes the environment name in to `meta.yml`. Ideally that would be passed through from the actual environment-specific job.

We have a pattern for keeping sensitive stuff out of pipeline configs now which is at https://github.com/dwp/aws-concourse/blob/master/bootstrap_ci_pipeline.py and https://github.com/dwp/aws-concourse/blob/master/ci/jobs/management-dev.yml.j2#L15